core/src/main/java/org/apache/calcite/sql/fun/SqlBitNotOperator.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to you under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.calcite.sql.fun;
+
+import org.apache.calcite.sql.SqlKind;
+import org.apache.calcite.sql.SqlPrefixOperator;
+import org.apache.calcite.sql.type.OperandTypes;
+import org.apache.calcite.sql.type.ReturnTypes;
+
+/**
+ * Defines the <code>BITNOT</code> prefix operator
+ *
+ * <p>Only INTEGER types are supported:
+ * tinyint, smallint, int, bigint
+ */
+public class SqlBitNotOperator extends SqlPrefixOperator {
+
+  SqlBitNotOperator() {
+    super("~", SqlKind.BITNOT, 0, ReturnTypes.ARG0_NULLABLE_IF_EMPTY,
```

Comment:
IMO, no need to create an extra class.

Reply:
Thank you @chunweilei . I have delete extra class.